pxc-80-backup/recovery-s3.sh

```diff
@@ -18,7 +16,7 @@ rm -rf /datadir/*
 xbcloud get "s3://${S3_BUCKET_URL}.sst_info" --parallel=10 | xbstream -x -C /datadir --parallel=$(grep -c processor /proc/cpuinfo)
 xbcloud get "s3://${S3_BUCKET_URL}" --parallel=10 | xbstream -x -C /datadir --parallel=$(grep -c processor /proc/cpuinfo)
 
-transition_key=$(parse_sst_info "/datadir/sst_info")
+transition_key=$(vault_get /datadir/sst_info)
 if [[ -n $transition_key ]]; then
     encrypt_prepare_options="--transition-key=\$transition_key"
 fi
```

Comment:
hide transition-key from logs
```suggestion
set +o xtrace
transition_key=$(vault_get /datadir/sst_info)
if [[ -n $transition_key ]]; then
    encrypt_prepare_options="--transition-key=\$transition_key"
    echo transition-key exists
fi
echo + xtrabackup ${XB_USE_MEMORY+--use-memory=$XB_USE_MEMORY} --prepare --binlog-info=ON --rollback-prepared-trx --xtrabackup-plugin-dir=/usr/lib64/xtrabackup/plugin --target-dir=/datadir
```